spec/active_record_data_store_spec.rb

```diff
@@ -294,6 +294,36 @@ def build_commit_id_map_from(phrase_arr)
     end
   end
 
+  describe '#each_unique_meta_key' do
+    it 'yields once for each unique meta key for the given repo' do
+      phrases = create_list(:phrase, 3, :with_meta_key)
+      dup_phrase = create(:phrase, meta_key: phrases.first.meta_key)
+
+      datastore.each_unique_meta_key(repo_name).to_a.tap do |phrases|
```

Comment:
Yeah cuz the test needs to check `phrases.size` and `phrases.uniq.size`, which have to be done on a full array version of the meta keys (not an Enumerable).